src/main/java/io/snyk/jenkins/SnykStepBuilder.java

```diff
@@ -210,20 +209,6 @@ public boolean perform(@Nonnull AbstractBuild<?, ?> build, @Nonnull Launcher lau
         throw new AbortException("Snyk API token with ID '" + snykTokenId + "' was not found. Please configure the build properly and retry.");
       }
       envVars.put("SNYK_TOKEN", snykApiToken.getToken().getPlainText());
-      envVars.overrideAll(build.getBuildVariables());
-
-      //workaround until we implement Step interface
-      VirtualChannel nodeChannel = node.getChannel();
-      if (nodeChannel != null) {
-        String toolHome = installation.getHome();
-        if (fixEmptyAndTrim(toolHome) != null) {
-          FilePath snykToolHome = new FilePath(nodeChannel, toolHome);
-          String customBuildPath = snykToolHome.act(new CustomBuildToolPathCallable());
-          envVars.put("PATH", customBuildPath);
-
-          LOG.info("Custom build tool path: '{}'", customBuildPath);
-        }
-      }
```

Comment:
I removed `snyk` entirely from my path and ran a build without this code and it works. So we don't need this.